internal/handlers/common/params.go

```diff
@@ -133,6 +135,11 @@ func GetWholeNumberParam(value any) (int64, error) {
 			return 0, errInfinity
 		}
 
+		if value > float64(math.MaxInt64) ||
+			value < float64(math.MinInt64) {
+			return 0, errLongExceeded
+		}
+
```

Comment:
There was a bug in the `GetWholeNumberParam` function, that returned negative `int64` values for positive `float64` exceeding `int64` range.

Reply:
wow nice catch!